### ISSUE TITLE
Question : How to access the authme player data?

### ISSUE THREAD

**Issue by DrOreo002** [AuthMe/AuthMeReloaded#1712]
I'm trying to make a custom plugin where it will shows how much accounts the player had on a GUI. So he can manage it or maybe see some information for the accounts

I've tried the **@Inject** method, like how the Accounts command do. But well It throws an NPE, so I try to find other way and I cannot found it, well there's a DataSource class declared on the AuthmeAPI but there's no getter for that variable on the class..., help?

**Comment by sgdc3** [comment]
Yeah we need a way to allow access to services handled by dependency injection.
